pom.xml

```diff
@@ -185,7 +185,7 @@
     <paranamer.version>2.8</paranamer.version>
     <maven-antrun.version>1.8</maven-antrun.version>
     <commons-crypto.version>1.0.0</commons-crypto.version>
-    <arrow.version>0.4.0</arrow.version>
+    <arrow.version>0.8.0-SNAPSHOT</arrow.version>
```

Comment:
Is there any ETA for the offficial 0.8.0?

Reply:
Can we download the snapshot from somewhere for our local tests?

Reply:
Should be able to cut an RC beginning of next week. I would suggest mvn-installing from Arrow master for the time being